tests/gold_tests/autest-site/microDNS.test.ext

```diff
@@ -68,7 +68,7 @@ def addRecords(self, records=None, jsonFile=None):
         f.write(json.dumps(jsondata))
 
 
-def MakeDNServer(obj, name, filename="dns_file.json", port=False, IP='127.0.0.1', rr=False, default=None, options={}):
+def MakeDNServer(obj, name, filename="dns_file.json", port=False, ip='127.0.0.1', rr=False, default=None, ipv6=False, options={}):
```

Comment:
This seems a bad API design choice to me. The family should be taken from the `ip` parameter, not specified separately. If you want to allow selection of special addresses by family, use the same technique as in C/C++ - reserve keywords such as "IP_LOOPBACK" or "IP_ADDR_ANY" to indicate such things.